### PR TITLE
Add https://github.com/mstuttgart/atty in project list

### PR DESCRIPTION
**Pull request by mstuttgart**
Changes proposed in this pull request:

- Add https://github.com/mstuttgart/atty to project list

## Before submitting

- [x] I've read and followed all steps in the [Making a pull request](https://github.com/allenai/python-package-template/blob/main/.github/CONTRIBUTING.md#making-a-pull-request)
    section of the `CONTRIBUTING` docs.
- [ ] I've updated or added any relevant docstrings following the syntax described in the
    [Writing docstrings](https://github.com/allenai/python-package-template/blob/main/.github/CONTRIBUTING.md#writing-docstrings) section of the `CONTRIBUTING` docs.
- [ ] If this PR fixes a bug, I've added a test that will fail without my fix.
- [ ] If this PR adds a new feature, I've added tests that sufficiently cover my new functionality.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +96,7 @@ Here is an incomplete list of some projects that started off with this template:
 - [beaker-py](https://github.com/allenai/beaker-py)
 - [gantry](https://github.com/allenai/beaker-gantry)
 - [ip-bot](https://github.com/abe-101/ip-bot)
+- [atyt](https://github.com/mstuttgart/atty)
```
Comment (epwalsh) [inline README.md:99]: typo?
```suggestion
- [atty](https://github.com/mstuttgart/atty)
```

Comment (mstuttgart) [reply README.md:99]: Nop.  It's `atty`: https://github.com/mstuttgart/atty

Comment (epwalsh) [reply README.md:99]: right... so, not "atyt"?

Comment (mstuttgart) [reply README.md:99]: > right... so, not "atyt"?

nop. It's `atty` from `a` lacri `tty` (alacritty terminal: https://github.com/alacritty)

Comment (mstuttgart) [reply README.md]: @epwalsh sorry. my fault hahah . Thanks to review.

**Review by epwalsh — APPROVED** [review]

**Pull request merged**